tietokanta/src/main/resources/db/migration/V1_870__.sql

```diff
@@ -0,0 +1,69 @@
+-- Vain osa toimenpidekooditaulun tehtävistä otetaan VEMTR- ja tehtämääräraporteilla mukaan
+-- Jotta tämä saadaan mahdollistettua, meidän pitää lisätä toimenpidekoodi tauluun merkintä,
+-- jossa yksiselitteisesti määritellään nämä tehtävät.
+-- Mukaanotettavat tehtävät on määritelty erillisessä excelissä.
+
+ALTER TABLE toimenpidekoodi
+    ADD "raportoi-tehtava?" BOOLEAN DEFAULT FALSE NOT NULL;
+
+-- Päivitetään raportoi-tehtava? = true, niiltä tehtäviltä, jotka halutaan noihin raportteihin
+-- Joudumme päivittämään nämä nimen perusteella, koska id ei täsmää eri ympäristöissä
+UPDATE toimenpidekoodi
+SET "raportoi-tehtava?" = TRUE
+WHERE (
```

Comment:
Itse kirjottelisin nämä varmasti muotoon WHERE nimi IN (a, b, c, d). Syntaksi hieman lyhenee.

Reply:
Hyvä pointti. Muutin.